src/graphql/root/subscription/price.ts

```diff
@@ -47,6 +50,14 @@ const PriceSubscription = {
     source: { errors: IError[]; satUsdCentPrice?: number },
     args: PriceResolveArgs,
   ) => {
+    if (source === undefined) {
```

Comment:
but the trace in comment is only from myUpdates... do you have a trace for this?

Reply:
No trace for this one, but I was thinking behaviour could maybe be the same at some point?